compare_lsmask_c384.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+import numpy as np
+from netCDF4 import Dataset
+
+ifname_gdas     = '/scratch2/NCEPDEV/stmp1/Zhichang.Guo/Work/Temp2/change_res/C384_ORIG/2025051421/mem001/sfc_data.nc'
```

Comment:
We should be avoiding variables hard-coded to someone's directory. 

Can this be turned into a user-specified variable?

Reply:
I agree. I was intending to provide an example script identifying the version of orog files rather than a generalized script as all versions of orog files are still hard coded and the strategy for identifying landmask and vegetation mask are dependent on the user-provided files. I will try to have it fixed.

Reply:
It's the restart file in your directory that should be generalized. I'm just going to leave this for now, and merge as is, since this is just an example script in case people need it.